### PR TITLE
Local

### PR DESCRIPTION
**Pull request by PHAGUN-JAIN**
binary search added 
whats new: 
fixed logical error added mid variable inside loop

### CODE REVIEW

File: Binary Search/binaryPJ.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
+
+ #include<iostream>
+ using namespace std;
+
+ int binary(int arr[], int n, int x){
+	int p=0;
+	int r=n-1;
+	
+ while(r>=p){
+	int m=(p+r)/2;
```
Comment (PHAGUN-JAIN) [inline Binary Search/binaryPJ.cpp:10]: here's the devil!

File: Binary Search/binaryPJ.cpp [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ int main(){
+	int arr[]={2,5,44,87,92};
+	int size=sizeof(arr)/sizeof(arr[0]);
+	int x=5;
+	int res=binary(arr, size, x);
+	if(res== -1){
+		cout<<"given no: \t"<< x <<"\tis not in the array"<<endl;
+	} else{
+		cout<<"the number is present at index:"<< res <<endl;
+		}
```
Comment (PHAGUN-JAIN) [inline Binary Search/binaryPJ.cpp:34]: previously were printing x in place of res

File: Binary Search/binaryPJ.cpp (hunk 1 above)
Comment (AkshatGoyal1018) [reply Binary Search/binaryPJ.cpp:10]: very good xD

File: Binary Search/binaryPJ.cpp (hunk 2 above)
Comment (AkshatGoyal1018) [reply Binary Search/binaryPJ.cpp:34]: Lol

**Pull request merged**